CHANGELOG.md

```diff
@@ -8,6 +8,21 @@
 
 ### Bug fixes
 
+## 1.8.3 (14 June 2018)
+
+### New features
+
+- Add an ISO 8601 DateTime scalar: `Types::ISO8601DateTime`. #1566
+- Use classes under the hood for built-in scalars. These are now accessible via `Types::` namespace. #1565
```

Comment:
I feel like this is more of a "maintenance" PR, but I preferred putting it under `New feature` than `Bug fixes` because it does expose new stuff.